static/assets/Rendering/CubismRenderer.ts

```diff
@@ -205,7 +205,7 @@ export default class CubismRenderer extends Component {
 
   //#region Material
   /** Material. */
-  @property({ type: Material, visible: true })
+  // @property({ type: Material, visible: true })
```

Comment:
This material appears to simply repost material from meshRenderer.
In the process of forwarding, initially this property is null, and then after the material on the meshRenderer is set to its normal value, there is data.
This results in a null being logged initially and an error in restoring.

After this modification, there will be no record problem, but it will not be displayed in the cubismRenderer component. Is this acceptable?

Reply:
@VisualSJ 
Acceptable.
It was determined that mesh should not necessarily be displayed.